src/status_im/ui/screens/profile/styles.cljs

```diff
@@ -12,9 +12,15 @@
             [status-im.utils.platform :as platform])
   (:require-macros [status-im.utils.styles :refer [defstyle]]))
 
+(defstyle toolbar-edit-text
+  {:padding-right 16
+   :color         styles/color-blue4
```

Comment:
We want to move to the new `colors` namespace (PR not yet merged)

Reply:
Awesome!